### PR TITLE
TFC and The Aether ct docs.

### PR DESCRIPTION
**Pull request by Mrthomas20121**
Here are Terrafirmacraft and The Aether ct docs.

### CODE REVIEW

File: docs/Mods/The_Aether/The_Aether.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# The Aether
```
Comment (kindlich) [inline docs/Mods/The_Aether/The_Aether.md:1]: Wanna split that into multiple files?

File: docs/Mods/The_Aether/The_Aether.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Package
+
+```zenscript
+import mods.aether_legacy.Accessory;
+```
+## Usage
+
+```zenscript
+Accessory.registerAccessory(IItemStack input, String accessory);
```
Comment (kindlich) [inline docs/Mods/The_Aether/The_Aether.md:13]: Any more info on what strings are accepted here?

File: mkdocs.yml [hunk 3]
```diff
@@ -815,6 +815,28 @@ nav:
...
+      - Terrafirmacraft: 'Mods/Terrafirmacraft/Terrafirmacraft.md'
+      - Heating: 'Mods/Terrafirmacraft/Heating.md'
+      - HeatingTemperatures: 'Mods/Terrafirmacraft/HeatingTemperatures.md'
+      - ItemRegistry: 'Mods/Terrafirmacraft/ItemRegistry.md'
+      - Metal: 'Mods/Terrafirmacraft/Metal.md'
+      - Recipes:
+        - Anvil: 
+          - Anvil: 'Mods/Terrafirmacraft/Recipes/Anvil/Anvil.md'
+          - Welding: 'Mods/Terrafirmacraft/Recipes/Anvil/Welding.md'
+        - Barrel: 'Mods/Terrafirmacraft/Recipes/Barrel.md'
```
Comment (kindlich) [inline mkdocs.yml:828]: You got Barrel added twice

File: docs/Mods/Terrafirmacraft/Recipes/Knapping/Stone.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+```zenscript
+StoneKnapping.removeRecipe(IItemStack output);
+StoneKnapping.removeRecipe(String registryName);
+```
+
+## Examples
+
+```zenscript
+// Gives a stone hoe in all rocks.
+StoneKnapping.addRecipe("testrecipe", [<minecraft:stone_hoe>], ["all"], "     ", "XXXX ");
```
[the remark this answers is not in the document]
Comment (Mrthomas20121) [reply docs/Mods/Terrafirmacraft/Recipes/Knapping/Stone.md:25]: done. https://github.com/CraftTweaker/CraftTweaker-Documentation/pull/285/commits/d26d6f97c859e0aa5ff654cb5a60384acb265f72

**Pull request merged**